package.json

```diff
@@ -81,7 +82,7 @@
     "tempy": "^1.0.0",
     "ts-node": "^10.5.0",
     "tstyche": "^1.0.0-beta.9",
-    "typescript": "^5.0.4",
```

Comment:
On a TS side, only `Symbol.dispose` is required, so it doesn't technically call for an upgrade of TS, but that property doesn't have types in previous versions of TypeScript, so I updated to TS 5.2.

Reply:
we're already on 5.3 in this repo. but this won't require our consumers to upgrade their TS version right (unless they use this feature)?